web/templates/popovers/left_sidebar_recent_view_popover.hbs

```diff
@@ -0,0 +1,11 @@
+<ul class="nav nav-list">
+    <li>
+        <a tabindex="0" class="set-default-view" data-view-code="{{view_code}}">
+            <i class="fa fa-home" aria-hidden="true"></i>
+            {{#tr}}
+                Make <z-view-name></z-view-name> my home view
+                {{#*inline "z-view-name"}}<b>{{t "recent conversation"}}</b>{{/inline}}
```

Comment:
I think you can just do `Make <strong>recent conversations</strong> my home view`; we do this pattern in many places.

Reply:
Done.